test/example/post/models.py

```diff
@@ -34,7 +34,7 @@ def save(self, *args, **kwargs):
                 _title = _title[:45]
             unique_slug = self.slug = slugify(_title)
             count = 1
-            while Post.objects.filter(slug=unique_slug).exists():
```

Comment:
This was wrong. Post is obviously not defined inside itself. Maybe because it never got executed, this was missed.

Reply:
I don't think this is wrong and Post is actually defined inside itself :). if your statement is true, I would not be able to create a post.